act-framework.bs

```diff
@@ -138,8 +138,86 @@ Editor note: This section describes the required properties of the data output b
 Rule Aggregation {#output-aggregation}
```

Comment:
Is this really in our purview to define how individual test results should be aggregated? Isn't the scope to just define the rule format and validation/benchmarking?